test/cpp/server/credentials_test.cc

```diff
@@ -20,34 +20,64 @@
 #include <grpcpp/security/server_credentials.h>
 #include <grpcpp/security/tls_credentials_options.h>
 #include <gtest/gtest.h>
+#include <stdio.h>
+#include <string.h>
 
 #include <memory>
 
 #include "src/cpp/client/secure_credentials.h"
 #include "test/core/util/port.h"
 #include "test/core/util/test_config.h"
+#include "include/grpc/grpc_security.h"
+#include "test/core/util/tls_utils.h"
+#include "src/core/lib/security/credentials/tls/grpc_tls_certificate_provider.h"
```

Comment:
I included "test/core/util/tls_utils.h" because I needd to use GetFileContents() to get the credentials I needed for testing. I will remove the other one, it's actually useless